src/UserStories/UserStories.css

```diff
@@ -157,18 +176,19 @@ html {
 .prev-button {
   grid-area: left;
   width: min-content;
-  margin-left: -50px;
+  margin-left: -46px;
```

Comment:
Seems like a magic number. I'd add a comment (similar to the JS doc we added in the JS file).

Reply:
On it